Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Grab the docker python image
+FROM ghcr.io/opensafely-core/python:latest as base-python
+
+# Upgrade pip and safetab
+RUN python -m pip install -U pip setuptools wheel && \
+    python -m pip install safetab-action
+
+# point to right file
+CMD python entrypoint.py
+
+# define entry point
+ENTRYPOINT ["safetab"]
```

Comment:
In both these, it would be good at test that the built docker file works. Sth like this in Makefile

```
IMAGE_NAME =? local-cohort-report
build:
    docker build . -t $(IMAGE_NAME)

test-docker: build
    docker run --rm -v tests/test_data:/workspace $(IMAGE_NAME) test_data_2.csv --config config.json
```
This will check your Dockerfile builds and you can run safetab.